positron/components/ModuleLoader.jsm

```diff
@@ -131,20 +144,22 @@ function ModuleLoader(processType) {
     }
 
     if (!file.exists()) {
-      throw new Error('No such module: ' + path);
+      throw new Error(`No such module: ${path} (required by ${requirer.id})`);
     }
 
     // dump('require: module found at ' + uri.spec + '\n');
 
     // Exports provided by the module.
-    let exports = Object.create(null);
+    let exports = Object.create({});
```

Comment:
What's the reason for using Object.create({}) here (instead of a normal object)?